content/_releases/22/32.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Product Update - v22.32
+description: Details of the product update v22.32 from 11th of August 2022.
+layout: rel
+version: "22.32"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-08-11
+redirect_from:
+  - /releases/22.32.html
+---
+
+
+## Improvements and Updates
+
+### Continued improvements
+
+Here we should talk about the automated vulnerabilities checks of our code.
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### Scalability of services
+
+elastic.io platform runs many services under the hood like `webhooks`, `api`, `frontend`, etc.
+Most of these services can and must run in replication meaning they can be scalled to suit
```

Comment:
Most of these services can be replicated, which allows the services to be scaled horizontally to address the required load and volume. However, some platform services are inherently not scalable. In this update we removed the possibility to define scaling within HELM chart configurations for 'bloodygate' and 'handmaiden' services.